lib/ansible/modules/packaging/os/installp.py

```diff
@@ -0,0 +1,347 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Kairo Araujo
+# Written by Kairo Araujo <kairo@kairo.eti.br>
+#
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: installp
+author: Kairo Araujo (@kairoaraujo)
+short_description: Installing packages for AIX
+description:
+  - Manage packages for AIX using 'installp'.
+version_added: "2.5"
```

Comment:
```suggestion
version_added: "2.8"
```
Should this be re-targeted for 2.8?

Reply:
Fixed.